cores/esp8266/FunctionalInterrupt.cpp

```diff
@@ -0,0 +1,24 @@
+#include <FunctionalInterrupt.h>
+
+
+// Duplicate typedefs from core_esp8266_wiring_digital_c
+typedef void (*voidFuncPtr)(void);
+
+// Helper functions for Functional interrupt routines
+extern "C" void ICACHE_RAM_ATTR __attachInterruptArg(uint8_t pin, voidFuncPtr userFunc, void*fp , int mode);
+
+// Structure for communication
+struct ArgStructure {
+	std::function<void(void)> reqFunction;
+};
+
+void interruptFunctional(void* arg)
```

Comment:
@igrr it's a bit late, but shouldn't this have the ICACHE_RAM_ATTR tag?

Reply:
Yes, you're right, under the current model it should have ICACHE_RAM_ATTR.